sky/clouds/gcp.py

```diff
@@ -916,6 +931,36 @@ def _check_instance_type_accelerators_combination(
             resources.instance_type, resources.accelerators, resources.zone,
             'gcp')
 
+    @classmethod
+    def check_disk_tier(
+            cls, instance_type: Optional[str],
+            disk_tier: Optional[resources_utils.DiskTier]) -> Tuple[bool, str]:
+        if disk_tier != resources_utils.DiskTier.ULTRA or instance_type is None:
+            return True, ''
+        # Ultra disk tier (pd-extreme) only support m2, m3 and part of n2
+        # instance types, so we failover to lower tiers for other instance
+        # types. Reference:
+        # https://cloud.google.com/compute/docs/disks/extreme-persistent-disk#machine_shape_support  # pylint: disable=line-too-long
```

Comment:
Does this mean we cannot use ultra disk tier with high end GPUs, such as A100, L4 or H100? In that case, this ultra disk sounds a bit useless here.

Reply:
Yes.. This is a quick fix to unblock the blog post first. We should think of better solution here. cc @Conless here